src/prefect/orion/schemas/core.py

```diff
@@ -1069,14 +1069,14 @@ class Artifact(ORMBaseModel):
         default=None,
         description="An identifier for how this artifact is persisted.",
     )
-    data: Optional[Any] = Field(
+    data: Dict[str, Any] = Field(
```

Comment:
I think this is a breaking change

Reply:
Oh sorry this `Artifact.data` not `State.data` — still breaking but perhaps not impactful

Reply:
Good note, reverted